Imports/ArcGIS/Runtime/Toolkit/Dialogs.2.0/qmldir

```diff
@@ -19,5 +19,8 @@ module Esri.ArcGISRuntime.Toolkit.Dialogs
 PortalItemsDialog 2.0 PortalItemsDialog.qml
 SimpleDialog 2.0 SimpleDialog.qml
 ImagePopUpDialog 2.0 ImagePopUpDialog.qml
-UserCredentialsDialog 2.0 UserCredentialsDialog.qml
-
+AuthenticationView 2.0 AuthenticationView.qml
+OAuth2View 2.0 OAuth2View.qml
+PkiCertificateView 2.0 PkiCertificateView.qml
```

Comment:
I don't see these 2 files in the PR: PkiCertificateView.qml and UntrustedHostView.qml did you miss adding them? Or may be remove these lines if you intend to add them later.